README.md

```diff
@@ -233,6 +235,27 @@ Need to setup Flatpak (https://flatpak.org/setup/):
 * Chrome OS
 
 
+Systemd
```

Comment:
Let's rename this to "Monthly run" or "Periodic run" or similar. And subcategory for Linux with systemd.

Reply:
Also let's save configs to `periodic/` directory instead of `systemd/`.

Reply:
Well this is somewhat a practice to put services into systemd directory, same as it used to for sysv or initscripts. I do not want anyone to be misguided by some 'periodic' or similar name.

Reply:
Systemd is just a way to implement periodic runs. Also we need to implement periodic runs for all supported systems in the same subcategory/directory (e.g. Linux w/o systemd, *BSD, etc.).

Reply:
ok makes sense, i'll update PR

Reply:
Done